front/app/containers/Admin/messaging/texting/CampaignList/TextCampaignListRow.tsx

```diff
@@ -70,45 +65,44 @@ const TextingCampaignRow = ({ campaign }: Props) => {
     attributes: { message, phone_numbers, status, sent_at },
   } = campaign;
 
+  const handleEvent = () => {
+    clHistory.push(`/admin/messaging/texting/${id}`);
+  };
+
   return (
-    <Container id={id}>
-      <Left>
-        <TextWrapper>
-          <Text>
-            <Link to={`/admin/messaging/texting/${id}`}>{message}</Link>
-          </Text>
-        </TextWrapper>
-        <FormattedStatusLabel campaignStatus={status} />
-      </Left>
-      <Right>
-        {status === 'sent' && (
-          <>
+    <Row onClick={handleEvent}>
+      <MessageCell>
+        <BodyText>{truncate(message, 60)}</BodyText>
+      </MessageCell>
+
+      {status === 'sent' && (
+        <>
+          <Cell>
             <DateTime>
```

Comment:
DateTime and SentText have the same styles. Could be 1 component + moved to the Parent component to simplify the component. You could call such a component `MetaInfoCell`.